### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by ironhide1975** [MisterPhilip/omnibug#150] [Ideas]
Can we get an option to remember the most recent tags we looked at in the filter. For example, I prefer to see just Adobe and Google at the top when I try to filter my view.

**Comment by MisterPhilip** [comment]
Just for clarification, are you suggesting that the order of the providers changes from alphabetical to most recently used in the filter popup? Are you adding these two providers every time you open Omnibug, and if so are you aware of the "Save as Default filters" option?
![image](https://user-images.githubusercontent.com/950141/127889399-12d62ed1-9d68-4ba5-9fae-9898b8d4c2d7.png)

**Reply by ironhide1975** [reply]
To Clarify:
What would be cool if, at the top, you could choose what were your favorite analytical packages. They would be automatically selected at the top. (say like top 5). 
Crude artist interpretation attached
![OmniBug Suggestion](https://user-images.githubusercontent.com/23507183/127931884-ac6d0709-cdd5-4a64-933b-e9322b5f81c6.jpg)

**Reply by MisterPhilip** [reply]
This is great, thanks for the suggestion! I am planning on adding profiles where you can automatically save what filters are enabled and some other new features to a profile (potentially automatically applied based on the URL/domain), would that suffice for your use case?

**Comment by ironhide1975** [comment]
That would be great if you could save filters for each domain.

**Comment by jcsia87** [comment]
Hi, we have lots of events generated in our pages, it's so tedious to have to collapse/expand and search through 50+ events manually. It would be cool to have a search text function and only show matching events.
